src/helix-ui/styles/core/hx-reveal.less

```diff
@@ -0,0 +1,7 @@
+hx-reveal {
+  display: none;
+
+  &[open] {
+    display: block;
+  }
+}
```

Comment:
Update: Modified the reveal Light DOM styling to remove crazy CSS needed when it still had a Shadow DOM.